README.md

```diff
@@ -1,2 +1,158 @@
-# docker-php
+# PHP-FPM
+
+[![Docker Stars](https://img.shields.io/docker/stars/spryker/php.svg)](https://store.docker.com/community/images/spryker/php)
+[![Docker Pulls](https://img.shields.io/docker/pulls/spryker/php.svg)](https://store.docker.com/community/images/spryker/php)
+[![microbadger version](https://images.microbadger.com/badges/version/spryker/php.svg)](https://microbadger.com/images/spryker/php "Get your own version badge on microbadger.com")
+[![microbadger image](https://images.microbadger.com/badges/image/spryker/php.svg)](https://microbadger.com/images/spryker/php "Get your own image badge on microbadger.com")
+
+# Description
+
 Extends official PHP Docker images with extensions and tools to be able to run Spryker on.
+
+* Based on: `Alpine 3.9`
```

Comment:
Technically our image is based on official PHP (which is based on Alpine 3.9).
Maybe we should mention that?

Reply:
The phrase above `Extends official PHP Docker images with extensions and tools to be able to run Spryker on.` already describes it.